src/Phrases.js

```diff
@@ -0,0 +1,15 @@
+export default class Phrases {
+    constructor(phrases) {
```

Comment:
we need to add a restriction to the number of phrases!
like throwing error, when phrases.length <= 2